src/main/java/blackjack/domain/participant/Participant.java

```diff
@@ -0,0 +1,70 @@
+package blackjack.domain.participant;
+
+import blackjack.domain.card.Card;
+import blackjack.domain.card.Deck;
+import blackjack.domain.card.Hand;
+import blackjack.domain.participant.rule.DrawRule;
+import java.util.List;
+
+public abstract class Participant {
+
+    protected static final int BLACKJACK_SCORE = 21;
+    private static final int START_CARDS_SIZE = 2;
+
+    private final Hand hand;
+
+    protected Participant(List<Card> cards) {
+        this.hand = new Hand(cards);
+    }
+
+    protected abstract int getMaxDrawableScore();
+
+    protected abstract int getStartCardSize();
+
+    protected abstract DrawRule getRules();
+
+    public final void drawStartCards(Deck deck) {
+        if (!hand.isEmpty()) {
+            throw new IllegalStateException("이미 시작 카드를 뽑았습니다.");
+        }
+        for (int i = 0; i < START_CARDS_SIZE; i++) {
+            add(deck.draw());
+        }
+    }
+
+    public final void add(Card card) {
+        if (!isDrawable()) {
+            throw new IllegalStateException("더 이상 카드를 추가할 수 없습니다.");
+        }
+        hand.add(card);
+    }
+
+    public void drawAdditionalCard(Deck deck) {
+        DrawRule rule = getRules();
+        rule.draw(this, deck);
+    }
+
+    public final boolean isDrawable() {
+        return calculateScore() <= getMaxDrawableScore();
+    }
+
+    public final int calculateScore() {
+        return hand.calculateScore();
+    }
+
+    public boolean isBlackjack() {
+        return hand.isBlackjack();
+    }
+
+    public final boolean isBusted() {
+        return hand.isBusted();
+    }
+
+    public final List<Card> getStartCards() {
+        return getCards().subList(0, getStartCardSize());
```

Comment:
`Hand`에게 메시지를 보내 볼까요? `getStartCards`를 추상 클래스로 두면 힌트가 보일지도 몰라요! 
```suggestion
        return hand.getStartCards()
```

Reply:
`Hand` 자체를 추상클래스로 둔다면 충분히 TDA를 지킬 수 있을 것 같네요.
그런데, 이 상황이라면 `Hand`를 다시 `DealerHand`와 `PlayerHand`로 분리시켜야 하며 구조가 조금 더 복잡해 질 것 같아요. 이 때의 이점이 "몇 장의 카드를 시작 카드로 보여줄 것인가?" 에 대한 추상화밖에 없다고 생각한다면 TDA를 지키는 것이 좋은 방향인가 고민이 되는 것 같네요.

추가적으로, TDA와 꽤나 관련이 깊은 "디미터의 법칙"의 경우 자료구조를 표현하기 위한 컬렉션에는 해당되지 않는다고 하는데요. `getStartCards()`의 반환값과 `getCards()`의 반환값이 모두 `List<Card>` 이기 때문에 저는 이 정도의 메서드 체인은 충분히 허용 가능하다고 생각해요.

Reply:
개인적으로는 `getStartCards()`를 추상화한 뒤에, 리턴 타입을 `List<Card>`로 하면 딜러는 한 장의 카드를, 플레이어는 두 장의 카드를 가져오도록 할 수 있을 것이라고 생각했어요. 이 정도만 해도 추상화하기에는 좋은 이점이라고 보았습니다 🏃🏻‍♂️ 결국 한 장을 보여줄지, 두 장을 보여줄지는 `Player`와 `Dealer`가 할 일이라고 생각했어요